BreweryRatings/Model/BusinessReview.cs

```diff
@@ -17,50 +17,66 @@ namespace ReviewNamespace
 
     public partial class BusinessReview
     {
+        private Review[] reviews;
+        private long total;
+        private string[] possibleLanguages;
+
         [JsonProperty("reviews")]
-        public Review[] Reviews { get; set; }
+        public Review[] Reviews { get => reviews; set => reviews = value; }
 
         [JsonProperty("total")]
-        public long Total { get; set; }
+        public long Total { get => total; set => total = value; }
 
         [JsonProperty("possible_languages")]
-        public string[] PossibleLanguages { get; set; }
+        public string[] PossibleLanguages { get => possibleLanguages; set => possibleLanguages = value; }
     }
 
     public partial class Review
     {
+        private string id;
+        private Uri url;
+        private string text;
+        private long rating;
+        private DateTimeOffset timeCreated;
+        private User user;
+
         [JsonProperty("id")]
-        public string Id { get; set; }
+        public string Id { get => id; set => id = value; }
 
         [JsonProperty("url")]
-        public Uri Url { get; set; }
+        public Uri Url { get => url; set => url = value; }
 
         [JsonProperty("text")]
-        public string Text { get; set; }
+        public string Text { get => text; set => text = value; }
 
         [JsonProperty("rating")]
-        public long Rating { get; set; }
+        public long Rating { get => rating; set => rating = value; }
 
         [JsonProperty("time_created")]
-        public DateTimeOffset TimeCreated { get; set; }
+        public DateTimeOffset TimeCreated { get => timeCreated; set => timeCreated = value; }
 
         [JsonProperty("user")]
-        public User User { get; set; }
+        public User User { get => user; set => user = value; }
     }
 
     public partial class User
     {
+        private string id;
+        private Uri profileUrl;
+        private Uri imageUrl;
+        private string name;
+
         [JsonProperty("id")]
-        public string Id { get; set; }
+        public string Id { get => id; set => id = value; }
```

Comment:
As I recall, 
        public string Id { get; set; }
		
is just shorthand for:
        public string Id { get => id; set => id = value; }
		
Same behavior; less typing.   If that's the case (and feel free to correct me if I'm wrong), I'd stick with less typing.